api/desecapi/models.py

```diff
@@ -334,6 +334,10 @@ def _partitioned_name(self):
         subname, _, parent_name = self.name.partition('.')
         return subname, parent_name or None
 
+    @property
+    def axfr(self):
```

Comment:
AXFR is the DNS Zone Transfer Protocol, RFC 5936. I don't think we should use that term here anywhere, because you can't "get" it, nor is it semantically clear would it would mean to "request https://.../domains/.../axfr" (one may expect a master-slave sync to happen?)

This is just getting the zone file in Bind format. This is different from AXFR format, which repeats the SOA record at the end to indicate "end of zone", for example. Maybe the clearest endpoint name would be `export`, and internally we can say `get_zonefile()` or something?

Reply:
`export` also allows to later return things in another format, based on `Accept`, for example. That also makes me think if one could not just do `GET .../rrsets/` with `Accept: text/dns` (MIME type taken from Wikipedia)?

Also, can we stream this from the name server? The zone file may be very large.

Reply:
If `GET` and `POST` should be possible, we could override `rrsets/`, or call it `zone` (I think that's better than `axfr`).

Reply:
Updated the name to be called "zonefile"; other suggestions warmly welcome. The ("outer") encoding is still JSON, maybe that offers better usability than messing with the Accept header?

Reply:
I think it's better to just return the plain zonefile as text/dns by default, that is when Accept: is missing or text/*. (I pushed some commits for inspiration). Perhaps we can return a JSON document when application/json is given, but I consider the plain export more important. (That's what all the media stuff in HTTP is about.)

OTOH, I don't feel strongly, so feel free to ignore. :)

Reply:
It's text/dns now without regard to the sent Accept header.